test/run_notebooks.sh

```diff
@@ -1,11 +1,39 @@
 #!/bin/bash
 
+RED='\033[0;31m'
+NC='\033[0m' # No Color
+
 echo "Running regression tests"
 
-for file in $(ls notebooks/*.ipynb); do
-  echo "$file"
-  dir="./output/$(basename ${file} .ipynb)"
-  mkdir -p "${dir}"
-  poetry run papermill "${file}" "${dir}/Results.ipynb" -p harmony_host_url $harmony_host_url
+# Add Docker images created in Makefile here
+images=(harmony harmony-regression)
+
+# launch all the docker containers and store their process IDs
+for image in ${images[@]}; do
+  PIDS+=(${image},$(docker run -d -v ${PWD}/output:/root/output -v ${PWD}/${image}:/root/${image} --env harmony_host_url="${HARMONY_HOST_URL}" "harmony/regression-tests-${image}:latest"))
+done
+
+# wait for processes to finish and store each exit code into array STATUS'
+for name_pid in ${PIDS[@]}; do
+  IFS=","
+  set -- $name_pid
+  name=$1
+  pid=$2
+  code=$(docker container wait ${pid})
+  EXIT_CODES+=(${code})
+  NAMES+=(${name})
+done
+
+# check to see if any of the docker runs errored
+i=0
+exit_code=0
+for code in ${EXIT_CODES[@]}; do
+  name=${NAMES[$i]}
+  if [[ ${code} -ne 0 ]]; then
```

Comment:
Add logs to note that the notebook is green, too, and possibly indicate that it has started.  Right now, on success, we get "Running regression tests", no output, a multi-minute-pause, then a zero exit code, which is very confusing and left me thinking something broke.